utils/merge_and_push_translations.sh

```diff
@@ -96,30 +96,16 @@ fi
 python3 utils/fix_formatting.py --lua --dir data/i18n
 python3 utils/fix_formatting.py --lua --dir data/txts
 
-# Undo one-liner diffs in po directory - these are pure timestamps with no other content
-if false; then  # Currently disabled due to TX no longer updating timestamps reliably
+# Undo one-liner diffs of pure timestamps with no other content
 set +x
-nrAdded=""
-nrDeleted=""
-for entry in $(git diff --numstat po); do
-  if [ -z "$nrAdded" ]
+for entry in $(git diff --numstat po/*/*.pot | sed -En 's/^1\t1\t//p'); do
+  if [ -z "$(git diff "$entry" | grep '^[+-][^+-]' | grep -v '^[+-]"POT-Creation-Date:')" ]
```

Comment:
or
```
sed -En -e '/^[^+-]/d' -e '/^[+-]([+-]|"POT-Creation-Date:)/d' -e p -e q
```
instead of the 2 greps, but I'm not sure if it's better

Reply:
IMHO `grep`s are far more readable than `sed` commands